### PR TITLE
Update pull request template

### PR DESCRIPTION
**Pull request by dennisameling**
See the attached file for details. Makes it clearer to folks which branch they should use for their PRs.

### CODE REVIEW

**Review by RCheesley — DISMISSED** [review]
GTG once 3.2 is released!

**Review by escopecz — DISMISSED** [review]
👍

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,9 +1,6 @@
-<!--
-Any PR related to Mautic 2 issues is not relavant anymore, please consider upgrading your code to the Mautic 3 series (staging/3.0 branch).
--->
 | Q                                      | A
 | -------------------------------------- | ---
-| Branch?                                | staging for features or enhancements / 3.2 for bug fixes <!-- see below -->
+| Branch?                                | "features" for features or enhancements / 3.2 for bug fixes <!-- see below -->
```
Comment (escopecz) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
| Branch?                                | "features" for features or enhancements / 3.2.1 for bug fixes <!-- see below -->
```
We can as well bump the bug fix version.

Comment (dennisameling) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: Wait... There won't be a branch for `3.2.1`, only a tag. The branch remains `3.2` for all 3.2.x related developments, right?.

Comment (escopecz) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: Oh, that make sense. I'm over-complicating things :) I'll delete the 3.2.1 and 3.3 branches for now.

**Review by RCheesley — APPROVED** [review]

**Pull request merged**